src/components/shared/MarkerTooltipContents.js

```diff
@@ -538,40 +554,45 @@ function getMarkerDetails(
                 'Redirect URL',
                 data.RedirectURI
               )}
-              {_markerDetail('pri', 'pri', data.pri)}
-              {_markerDetailNullable('count', 'count', data.count)}
+              {_markerDetail('pri', 'Priority', data.pri)}
```

Comment:
Not for this PR: I wonder if we can get some human readable values here, as opposed to random numeric values. I just searched through the [gecko code and it appears that this is actually an enum value](https://searchfox.org/mozilla-central/rev/a80651653faa78fa4dfbd238d099c2aad1cec304/xpcom/threads/nsISupportsPriority.idl#24-28). I think it would be better to get this value as a string. I'll ask Randell about it.

Reply:
I filed https://github.com/devtools-html/perf.html/issues/1130 after chatting with Randell

Reply:
Perfect, thank you! :)